### PR TITLE
Add the 2022 Matrix Community Year In Review

### PR DESCRIPTION
**Pull request by deepbluev7**
Open questions:

- I can't really get the summary tags to work as I wanted to
- Should I resize the images? :D

Preview: https://pr1600--matrix-org-previews.netlify.app

### CODE REVIEW

File: gatsby/content/blog/2023/01/2023-01-03-matrix-community-year-in-review.mdx [hunk 1]
```diff
@@ -0,0 +1,676 @@
...
+title: 'Matrix Community Year In Review 2022'
+categories:
+  - General
+author: Nico
+image: https://matrix.org/blog/img/matrix-logo.png
+---
+
+> Note: This was originally posted on https://blog.neko.dev/posts/matrix-year-in-review-2022.html , which also includes some small info boxes about each projects, which got lost in the translation.
+
+As we send off 2022 with a bang, it makes sense to look back on what we did this year and where we want to go next year. Usually the Matrix organization posts such an update, but I felt like this years update was a bit scary and also light on community projects.
```
Comment (thibaultamartin) [inline gatsby/content/blog/2023/01/2023-01-03-matrix-community-year-in-review.mdx:12]:
```suggestion
As we send off 2022 with a bang, it makes sense to look back on what we did this year and where we want to go next year. In its holiday special post, the Matrix Foundation has been focusing on the core team's work. This post is focusing on the achievements of the community outside of the Matrix Foundation.

I tried to reach out to as many people I have seen do "stuff" on Matrix and have them write something they would see fitting for a year in review. Now, most people have better things to do between christmas and new years, so I hope you can excuse if some projects are missing. Also I probably forgot like half of the interesting people... HOWEVER! I hope you still enjoy what everyone wrote up. And don't forget to check out the official [Matrix Holiday Update 2022](https://matrix.org/blog/2022/12/25/the-matrix-holiday-update-2022) and of course read you weekly [TWIM](https://matrix.org/blog/category/this-week-in-matrix) to keep up to date with any cool projects you find.
```

**Review by thibaultamartin — CHANGES_REQUESTED** [review]
Minor nit on the intro :)

File: gatsby/content/blog/2023/01/2023-01-03-matrix-community-year-in-review.mdx (hunk 1 above)
Comment (deepbluev7) [reply gatsby/content/blog/2023/01/2023-01-03-matrix-community-year-in-review.mdx:12]: Yeah, that sounds way better, but I think your quote is misplaced.

File: gatsby/content/blog/2023/01/2023-01-03-matrix-community-year-in-review.mdx [hunk 2]
```diff
@@ -0,0 +1,676 @@
...
+
+So to improve on that, I tried to reach out to as many people I have seen do "stuff" on Matrix and have them write something they would see fitting for a year in review. Now, most people have better things to do between christmas and new years, so I hope you can excuse if some projects are missing. Also I probably forgot like half of the interesting people... HOWEVER! I hope you still enjoy what everyone wrote up. And don't forget to check out the official [Matrix Holiday Update 2022](https://matrix.org/blog/2022/12/25/the-matrix-holiday-update-2022) and of course read you weekly [TWIM](https://matrix.org/blog/category/this-week-in-matrix) to keep up to date with any cool projects you find.
```
Comment (deepbluev7) [inline gatsby/content/blog/2023/01/2023-01-03-matrix-community-year-in-review.mdx:14]:
```suggestion
As we send off 2022 with a bang, it makes sense to look back on what we did this year and where we want to go next year. In its holiday special post, the Matrix Foundation has been focusing on the core team's work. This post is focusing on the achievements of the community outside of the Matrix Foundation.
I tried to reach out to as many people I have seen do "stuff" on Matrix and have them write something they would see fitting for a year in review. Now, most people have better things to do between christmas and new years, so I hope you can excuse if some projects are missing. Also I probably forgot like half of the interesting people... HOWEVER! I hope you still enjoy what everyone wrote up. And don't forget to check out the official [Matrix Holiday Update 2022](https://matrix.org/blog/2022/12/25/the-matrix-holiday-update-2022) and of course read you weekly [TWIM](https://matrix.org/blog/category/this-week-in-matrix) to keep up to date with any cool projects you find.
```

Comment (thibaultamartin) [reply gatsby/content/blog/2023/01/2023-01-03-matrix-community-year-in-review.mdx]: LGTM

**Pull request merged**